### ISSUE TITLE
2nd Community Review of 1475 EFil+ Allocator

### ISSUE THREAD

**Issue by filecoin-watchdog** [filecoin-project/Allocator-Governance#143]
First Review https://github.com/filecoin-project/Allocator-Governance/issues/57

Allocator Compliance Report: https://compliance.allocator.tech/report/f03018491/1724113418/report.md

2.5PiBs DataCap awarded in 2nd round

example 1: https://github.com/1475Notary/1475-Allocator/issues/12
example 2: https://github.com/1475Notary/1475-Allocator/issues/5
example 3: https://github.com/1475Notary/1475-Allocator/issues/14

**Comment by Kevin-FF-USA** [comment]
Hi @1475Notary,

Wanted to ensure you were tracking this refill issue for 1475's EFil

**Comment by 1475Notary** [comment]
@Kevin-FF-USA Thanks Kevin. We're on it.

**Comment by 1475Notary** [comment]
After granted last time, we allocate datacap to 3 clients, one of which was a new client.

- [issue14](https://github.com/1475Notary/1475-Allocator/issues/14)

We review our clients before allocating datacap.
![image](https://github.com/user-attachments/assets/10b4e62e-f894-4ec5-9dde-27d406b7f19c)

- [issue5](https://github.com/1475Notary/1475-Allocator/issues/5)

After the client updated their plan in a timely manner, I allocated a new round of datacap for the client.
![image](https://github.com/user-attachments/assets/d66db3df-9913-4dbd-96ff-40f4daf442af)

- [issue12](https://github.com/1475Notary/1475-Allocator/issues/12)

The first round of datacap was allocated after we did some check to this client.
![image](https://github.com/user-attachments/assets/f86b7f85-fbaa-49f3-83db-d3498e260112)
After confirming that the client had updated their information in time, we allocated a second round datacap.

Retrieval is highly valued by us. After communication with clients, they stopped working with sps that didn't support retrieval. Some nodes even reached a 100 percent successful retrieval rate.

We will continue to review our clients' process and help them to improve their data storage.

**Comment by galen-mcandrew** [comment]
This is the second compliance review of this manual allocator, and there is still mixed performance overall. Specifically:

- Improving retrieval of public open datasets, average retrievability success rate of 50%
- High usage of VPNs without enforcing stated requirements from allocator application (https://github.com/filecoin-project/notary-governance/issues/1047 "We will do KYB checks on each SP using VPN"
- Noncompliant SP distribution (Data Distribution section of application), with continued subsequent allocations (such as https://check.allocator.tech/report/1475Notary/1475-Allocator/issues/5/1725660605422.md)
- Mixed KYC diligence into client claims

As a reminder, the allocator team is responsible for verifying, supporting, and intervening with their clients. If a client is NOT providing accurate deal-making info (such as incomplete or inaccurate SP details) or making deals with noncompliant unretrievable SPs, then the allocator needs to intervene and require client updates before more DataCap should be awarded.

Overall, we are seeing evidence of allocator diligence and interventions. We are requesting an **additional 5PiB of DataCap from RKH**, to allow this allocator to show increased diligence and alignment.

Please verify that you will instruct, support, and require your clients to work with retrievable storage providers. @1475Notary  can you verify that you will enforce retrieval requirements, such as through Spark? Additionally, we would still like to see the progress on the smart contract development.

**Comment by 1475Notary** [comment]
Yes, we verify that we will do better in the next step, especially retrieval. And we would give more advices to clients about sp and data distribution.
All in process and need more time.
Thank you @galen-mcandrew

**Comment by Kevin-FF-USA** [comment]
DataCap Refreshed: https://filecoinpulse.pages.dev/allocator/f03018491/#pageparamsallocator_id
Closing Issue
